### DISCUSSION TITLE
What exactly is installed/not installed using the minimal installer feature?

### DISCUSSION THREAD

**Discussion by rexbinary** [ElPumpo/TinyNvidiaUpdateChecker#206] [Q&A] [answered]
Hi all

I am unclear what is installed and what is not installed when using the minimal installer feature. If I missed this information somewhere I apologize in advance.

I know the drivers are installed, but what about audio drivers, PhysX, Ansel, and the nvidia control panel?

**Comment by HawaiiBeach** [comment, marked as answer]
Hi! You are correct, the information about this feature is not documented properly in the wiki. I will soon add the missing info.

Currently it will only install the GPU driver and nothing else with it. There is a bug report for making it user customizable

**Comment by rexbinary** [comment]
Thank you!
